lib/common/utils.js

```diff
@@ -21,6 +21,12 @@ export function getClientArchVersion (arch) {
   }
 }
 
+export function omit (obj, keys) {
+  return Object.keys(obj)
+    .filter(k => !keys.includes(k))
+    .reduce((a, k) => ({ ...a, [k]: obj[k] }), {});
+}
```

Comment:
This would be more efficient being implemented as a single loop, and having a `result` object that it mutates instead of cloning it for each key.

Is this function used anywhere?

Reply:
I use it mostly for debugging tests, should be useful to leave it there, will optimize in case we do add to other code.